### ISSUE TITLE
OpenShift connector plugin doesn't work

### ISSUE THREAD

**Issue by vinokurig** [eclipse-che/che#16261]
### Describe the bug

The OpenShift Connector plugin's view is showed but commands don't work
### Che version

 - [ ] latest
 - [x] nightly
 - [ ] other: please specify

### Steps to reproduce
1. Start a workspace with OpenShift connector plugin enabled
2. Execute the `Login` command
See: Error in the browser's console:
```
theia.27611f06b4b4ec17f8c5.js:1 Uncaught (in promise) Error: command 'openshift.explorer.login' not found
    at e.<anonymous> (theia.27611f06b4b4ec17f8c5.js:1)
    at theia.27611f06b4b4ec17f8c5.js:1
    at Object.next (theia.27611f06b4b4ec17f8c5.js:1)
    at theia.27611f06b4b4ec17f8c5.js:1
    at new Promise (<anonymous>)
    at o (theia.27611f06b4b4ec17f8c5.js:1)
    at Object.execute (theia.27611f06b4b4ec17f8c5.js:1)
    at e.<anonymous> (theia.27611f06b4b4ec17f8c5.js:1)
    at theia.27611f06b4b4ec17f8c5.js:1
    at Object.next (theia.27611f06b4b4ec17f8c5.js:1)
```

### Expected behavior

The login command must display a login input
### Runtime
   - [ ] kubernetes (include output of `kubectl version`)
   - [ ] Openshift (include output of `oc version`)
   - [ ] minikube (include output of `minikube version` and `kubectl version`)
   - [x] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Installation method
   - [x] chectl 
   - [ ] che-operator 
   - [ ] minishift-addon
   - [ ] I don't know

### Environment
   - [ ] my computer
       - [ ] Windows
       - [x] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [ ] other: please specify

**Comment by nickboldt** [comment]
We should compare what's happening in this che/registry/plugin/sidecar combination (need all those versions) vs. what we do in CRW 2.1 to decide if this is a blocker for 2.1 / 7.9.2, or just for 7.10.

CRW 2.1 side car image: http://pkgs.devel.redhat.com/cgit/containers/codeready-workspaces-plugin-openshift/tree/Dockerfile?h=crw-2.0-rhel-8

CRW 2.1 plugin reg: https://github.com/redhat-developer/codeready-workspaces/blob/master/dependencies/che-plugin-registry/v3/plugins/redhat/vscode-openshift-connector/0.1.4/meta.yaml

Che 7.x side car image:
* https://github.com/che-dockerfiles/che-sidecar-openshift-connector/blob/0.1.2/Dockerfile
* https://github.com/che-dockerfiles/che-sidecar-openshift-connector/blob/0.1.4/Dockerfile (empty)

Che 7.9.x plugin reg: https://github.com/eclipse/che-plugin-registry/tree/master/v3/plugins/redhat/vscode-openshift-connector (latest is 0.1.2, not 0.1.4)

Stalled PRs to move to 0.1.4:
* https://github.com/eclipse/che-plugin-registry/pull/397
* https://github.com/che-dockerfiles/che-sidecar-openshift-connector/pull/7#pullrequestreview-359160099

So... maybe this is fixed by moving up to 0.1.4, as we did **9 days ago** in CRW 2.1?

**Comment by azatsarynnyy** [comment]
We were investigating this issue on @vinokurig minishift installation and couldn't reproduce it anymore after restarting his minishift/Che. Probably, it was related to a broken minishift/Che installation.
Also, we could not reproduce it either on my minishift or Hosted Che.
So, @vinokurig please close the issue if the plugin works well for you.
